### ISSUE TITLE
Bits editor should respect asm.bits

### ISSUE THREAD

**Issue by brainstorm** [radare/radare2#5819]
The current incantation of radare's bit editor does not respect the length of the instruction (`asm.bits`). See radare's screenshot vs datasheet opcode bits:

![skarmavbild 2016-09-21 kl 09 48 16](https://cloud.githubusercontent.com/assets/175587/18713779/cbd27b38-8013-11e6-9f3d-1cf6414ec490.png)
![skarmavbild 2016-09-21 kl 09 49 37](https://cloud.githubusercontent.com/assets/175587/18713777/cbce7c68-8013-11e6-81be-38b98a45c1b5.png)

In other words, in the bit view of radare, it would be great to have just **2 bytes of the current opcode** (`lpm` in this case), greying out or discarding all the surrounding bits.

Other crazy useful addons suggested by @radare would be adding keybindings for **bit shifting** and **bit rotations** in the bit editor.

**Comment by radare** [comment]
Asm.bits is honored here too. Whats the problem?

**Comment by radare** [comment]
this is reflected in `hex`.. not sure if the bit editor will work on > 8 byte words

**Comment by radare** [comment]
also, add a seaprate issue for the bitshifting/rotation stuff

**Comment by brainstorm** [comment]
Solved in https://github.com/radare/radare2/commit/43a3eacd :)
